src/node/mini_miner.h

```diff
@@ -90,10 +100,7 @@ class MiniMiner
     // the same tx will have the same bumpfee. Excludes non-mempool transactions.
     std::map<uint256, std::vector<COutPoint>> m_requested_outpoints_by_txid;
 
-    // Txid to a number representing the order in which this transaction was included (smaller
-    // number = included earlier).  Transactions included in an ancestor set together have the same
-    // sequence number.
-    std::map<Txid, uint32_t> m_inclusion_order;
+    LinearizationResult m_linearize_result;
```

Comment:
In  [mini miner]: Linearize should also return packages fees and sizes:

Would be nice to keep a comment about this member I think.

Reply:
It's preserved in the new struct, all members of `m_linearize_result` were described.